xla/myplugin/myplugin_pjrt.cc

```diff
@@ -0,0 +1,46 @@
+#include "xla/pjrt/c/pjrt_c_api_wrapper_impl.h"
+#include "myplugin_pjrt.h"
+#include "myplugin_cpp_pjrt.h"
+
+namespace myplugin_pjrt {
```

Comment:
Should filename be `myplugin_c_pjrt`? Looks like this is the capi impl?

Reply:
Moved.